main.py

```diff
@@ -11,28 +14,34 @@
 BULLETS_VELOCITY = 10
 RED = (255, 0, 0)
 
-SPACESHIP_IMAGE = pygame.image.load(os.path.join("Assets", "spaceship_red.png"))
-SPACESHIP = pygame.transform.rotate(
-    pygame.transform.scale(SPACESHIP_IMAGE, (SPACESHIP_WIDTH, SPACESHIP_HEIGHT)), 180
+SPACESHIP = Player(
+    "Player1",
+    "spaceship_red.png",
+    SPACESHIP_WIDTH,
+    SPACESHIP_HEIGHT,
+    angle=180,
+    hp=100,
 )
 
-ENEMY_IMAGE = pygame.image.load(os.path.join("Assets", "spaceship_yellow.png"))
-ENEMY = pygame.transform.rotate(
-    pygame.transform.scale(ENEMY_IMAGE, (SPACESHIP_WIDTH, SPACESHIP_HEIGHT)), 0
-)
 
-SPACE_IMAGE = pygame.image.load(os.path.join("Assets", "space.png"))
-SPACE = pygame.transform.rotate(
-    pygame.transform.scale(SPACE_IMAGE, (WIDTH, HEIGHT)), 180
+ENEMY = Player(
+    "Enemy1",
+    "spaceship_yellow.png",
+    SPACESHIP_WIDTH - 3,
+    SPACESHIP_HEIGHT - 3,
+    angle=0,
+    hp=50,
 )
 
+SPACE = load_png("space.png", WIDTH, HEIGHT, 0)
+
 # ENEMY_HIT = pygame.USEREVENT
 
 
 def draw_window(spaceship, player_bullets, enemy):
-    WINDOW.blit(SPACE, (0, 0))
-    WINDOW.blit(SPACESHIP, (spaceship.x, spaceship.y))
-    WINDOW.blit(ENEMY, (enemy.x, enemy.y))
+    WINDOW.blit(SPACE[0], (0, 0))
```

Comment:
load_png returns a tuple, the first object in the tuple is a Surface which is what the blit function expects as input.